packages/did/src/Did.signature.spec.ts

```diff
@@ -139,10 +140,11 @@ describe('light DID', () => {
     })
     keyUri += '1a'
     jest.mocked(resolveKey).mockRejectedValue(new Error('Key not found'))
-    await expect(() =>
+    await expect(
       verifyDidSignature({
         message: SIGNED_STRING,
         signature,
+        // @ts-ignore Appending the uri has changed its type
```

Comment:
Would it make more sense to rather cast the URI type above? I am not a fan of ts-ignore and ts-expect-error…

Reply:
For me personally it would be preferable

Reply:
Oh yeah, the template string is even better!

Reply:
props to eslint :D